File: safe_place/v1/by_id/python/safe_place.py
# Install the Python library from https://pypi.org/project/amadeus
from amadeus import ResponseError, Client

amadeus = Client(
    client_id='YOUR_AMADEUS_API_KEY',
    client_secret='YOUR_AMADEUS_API_SECRET'
)

try:
    '''
    Returns safety information in Barcelona within a designated area
    '''
    response = amadeus.safety.safety_rated_location('Q930400801').get()
    print(response.data)
except ResponseError as error:
    raise error

File: safe_place/v1/by_id/node/safe_place.js
var Amadeus = require("amadeus");
var amadeus = new Amadeus({
  clientId: 'YOUR_API_KEY',
  clientSecret: 'YOUR_API_SECRET'
});

// What is the safety information of a location based on it's Id?
amadeus.safety.safetyRatedLocation('Q930400801').get().then(function (response) {
  console.log(response);
}).catch(function (response) {
  console.error(response);
});